cmd/rdiff/delta.go

```diff
@@ -1,8 +1,63 @@
 package main
 
 import (
+	"io/ioutil"
+	"os"
+
+	"github.com/balena-os/librsync-go"
+	"github.com/sirupsen/logrus"
 	"github.com/urfave/cli"
 )
 
+var (
+	magic            = librsync.BLAKE2_SIG_MAGIC
+	blockLen  uint32 = 512
+	strongLen uint32 = 32
+	bufSize          = 65536
```

Comment:
We should get these values from the signature file. See my other comment below.

Reply:
I removed it